packages/calypso-products/src/constants/akismet.ts

```diff
@@ -22,6 +22,18 @@ export const PRODUCT_AKISMET_ENTERPRISE_2M_YEARLY = 'ak_ep2m_yearly_1';
 export const PRODUCT_AKISMET_ENTERPRISE_2M_MONTHLY = 'ak_ep2m_monthly_1';
 export const PRODUCT_AKISMET_ENTERPRISE_GT2M_YEARLY = 'ak_epgt2m_yearly_1';
 export const PRODUCT_AKISMET_ENTERPRISE_GT2M_MONTHLY = 'ak_epgt2m_monthly_1';
+export const PRODUCT_AKISMET_PRO_MONTHLY = 'ak_pro5h_monthly';
```

Comment:
What do you think about including the API count in the constant name here?

`PRODUCT_AKISMET_PRO_500_MONTHLY`

Reply:
Good thought, I will make that change